common/primitives/log.go

```diff
@@ -12,6 +12,8 @@ import (
 	"strconv"
 )
 
+// Log prints the formated string with args (using fmt.Printf) with a prepended file and line number of
+// the calling function: "<file>:<linenumber> - <fmt.Printf_output>\n"
 func Log(format string, args ...interface{}) {
```

Comment:
All this log stuff isn't used. We changed how we log, so honestly we could delete this file

Reply:
I'd be in favor of deleting the functions if they are no longer used anywhere or relevant. But I let you guys make that determination.